cmd/scaffold.go

```diff
@@ -204,7 +205,8 @@ func (fnb *FlowNodeBuilder) enqueueNetworkInit() {
 		fnb.Network = net
 
 		idRefresher := libp2p.NewNodeIDRefresher(fnb.Logger, fnb.State, net.SetIDs)
-		fnb.ProtocolEvents.AddConsumer(idRefresher)
+		idEvents := gadgets.NewIdentityDeltas(idRefresher.OnIdentityTableChanged)
+		fnb.ProtocolEvents.AddConsumer(idEvents)
```

Comment:
I kept `IdentityDeltas` separate because I think it's likely useful independent of the networking layer. But I agree this is stacking two very thin layers. I'll take another pass to see if I can simplify this.

Reply:
https://github.com/onflow/flow-go/pull/143/commits/0e2f3c9b7d4c0b9f5b49e5cc312d8c0d74838e47